en-US/win10/samples/arduino-wiring/DCMotor.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: default
+title: Arduino Wiring DC Motor Sample
+permalink: /en-US/win10/samples/arduino-wiring/DCMotor.htm
+lang: en-US
+---
+
+#Arduino Wiring DC Motor Sample
+
+DC motors are some of the basic elements of a maker's toolbox. Whether you're driving autonomous robots or spinning a cooling fan, a DC motor is a relatively simple way to accomplish your task! In this sample, we'll use pulse-width modulation (PWM) to vary the speed of a motor.
```

Comment:
Maybe change opening line to "DC motors are a basic element of a maker's toolbox."

"DC Motors" as a phrase seems like its meant to refer to it as a singular idea, not having many DC motors.